script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -430,6 +430,7 @@ The figure's laugh echoes through the darkness.
 -- Page 507, Offset 21032.
 -- ぎょろりと血走ったその瞳が“ようやく気が付いたのか”と嘲笑している。
 // Dude can you see him or not
+// group pr note: eye/eyes?
```

Comment:
I think that plural is OK here, it initially only mentions one visible eye but there's an animation that plays between that and now where 2 eyes are visible. Plus, if it's SHIKI+Roa, that sprite has 2 eyes one is just covered somewhat by hair